### DISCUSSION TITLE
RPITIT

### DISCUSSION THREAD

**Discussion by Cobord** [Quantinuum/hugr#886] [Q&A] [answered]
In forked repo https://github.com/cobord/hugr, have changes to utilize RPITIT as of version 1.75. Don't want to PR this directly to main because it would cause breaking changes to others that have hugr as a dependency (corresponding changes in https://github.com/Cobord/tket2/ that would be done simultaneously and similar for others).

**Comment by aborgna-q** [comment, marked as answer]
Hi. We are planning to do a breaking release next (`0.3.0`), so now it's the right time to merge in some refactor.

Feel free to make a PR here. The `tket2` one will have to wait until we update the hugr dependency over there, but you can create it as a draft.
